examples/EvaluationDataToApplicationInsights/Pages/Index.cshtml

```diff
@@ -0,0 +1,10 @@
+﻿@page
+@model IndexModel
+@{
+    ViewData["Title"] = "Home page";
+}
+
+<div class="text-center">
+    <h1 class="display-4">Welcome <script>document.write(getCookie("username"));</script></h1>
```

Comment:
Given this is a razor page, I'd recommend putting the username into the view data and using razor syntax to inject it into the DOM.